### DISCUSSION TITLE
Is different gear possible?

### DISCUSSION THREAD

**Discussion by provisionforvoids** [OpenTracksApp/OpenTracks#1816] [Q&A]
In some commercial sport tracker applications you can choose which gear you used for the activity. For example, I have two different road bikes that both are in use. It would be nice to be able to see which bike I used for which ride and how many rides/hours/distance I did with each bike say in a year.

Only way I came up this can kind of be done, is to manually edit the activity type. So when you go to see the aggregated stats, instead of biking, the activity type will be for example "Trek" or "Specialized" etc. But IMO this is not an ideal solution, because the activity type must be manually edited every time, so if there is a spelling mistake, it will add a completely new type of activity.

**Comment by dennisguse** [comment]
This feature is tracked here: #395 
To the best of my knowledge there was no work done on that.
But you are right, it would be nice to have.
